edisgo/grid/network.py

```diff
@@ -2064,6 +2075,62 @@ def grid_expansion_costs(self):
     def grid_expansion_costs(self, total_costs):
         self._grid_expansion_costs = total_costs
 
+    @property
+    def grid_losses(self):
+        """
+        Holds the losses in the grid obtained from the slack bus in
+        kW and kvar.
+
+        Returns
+        -------
+        :pandas:`pandas.DataFrame<dataframe>`
```

Comment:
What are column names and index of the df?

Reply:
The correction will be pushed with the PR of branch test/curtailment_results